examples/sb3/scenarios/single_agent_tasks/junction_left_turn/scenario.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+from smarts.sstudio.types import Distribution
+from smarts.sstudio import gen_scenario
+from smarts.sstudio import types as t
+
+flow1 = [t.Flow(
+            route=t.Route(
+                begin=("-E3",0, "random"), end=("-E0",0, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        )
+        for _ in range(1)]
+
+flow2 = [t.Flow(
+            route=t.Route(
+                begin=("E0",0, "random"), end=("E3",0, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        )
+        for _ in range(1)]
+
+flow3 = [t.Flow(
+            route=t.Route(
+                begin=("E4", lane, "random"), end=("E1", lane, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        ) for lane in range(2)
+        for _ in range(2)]
+
+flow4 = [t.Flow(
+            route=t.Route(
+                begin=("-E1", lane, "random"), end=("-E4", lane, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        ) for lane in range(2)
+        for _ in range(2)]
+
+flow5 = [t.Flow(
+            route=t.Route(
+                begin=("E0", 0, "random"), end=("E1", 1, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        )
+        for _ in range(1)]
+
+
+traffic = t.Traffic(
+    flows=flow1+flow2+flow3+flow4+flow5
+)
+
```

Comment:
1. Other than the number of lanes in the horizontal road, how does this left-turn scenario differ from that in the pending PR #1274?
2. Was this scenario intended to be unprotected-left-turn or left-turn-with-traffic-lights? The junction in this scenario appears to have traffic lights.

Reply:
Re:
1. Just checked #1274, I think they are pretty similar, only the lane number and traffic density are different. This scenario was created from this figure below. There are more similar scenarios with different lane numbers and traffic densities (as sub-tasks) will be created as Soheil requested. 
![image](https://user-images.githubusercontent.com/59240680/166268613-39c69cc1-b2a8-4db6-abc7-f5a63663782b.png)
2. I temporarily put a traffic light in the intersection just for making the scenario look smoother, otherwise there is traffic jam due to the heavy traffic (some vehicles would never have a chance to turn or pass). I can easily remove the traffic light if needed.

Reply:
yeah it should, I left it undone since I was not sure how the mission should look like. I'll confirm with Soheil and Tucker and add it soon.